web/src/hooks/useDraftTask.tsx

```diff
@@ -50,24 +59,24 @@ export default function useDraftTask(
   return [draft, actions];
 }
 
-function useDraftAssignee(draft: DraftTask | null): User | undefined {
-  const [draftAssignee, setDraftAssignee] = React.useState<User | undefined>(
-    undefined
-  );
-  const [getUserQuery] = useGetUserLazyQuery({
-    onCompleted: ({ user }: GetUserQuery) => {
-      console.log("user", user);
-      if (user) {
-        setDraftAssignee(user);
-      }
-    },
-  });
+// function useDraftAssignee(draft: DraftTask | null): User | undefined {
```

Comment:
Question: Is this meant to be commented out?

Reply:
Oops! I simplified the logic here so this can just go entirely, nice catch!